lua/render-markdown/init.lua

```diff
@@ -244,6 +246,9 @@ M.default_config = {
         style = 'full',
         -- Amount of padding to add to the left of code blocks
         left_pad = 0,
+        -- Amount of padding to add to the right of code blocks
+        -- when width is 'block'
+        right_pad = 1,
```

Comment:
Can this be `0` by default? Since `block` width is opt in I'm not sure if a default makes sense, but open to either.

Reply:
updated